dask_jobqueue/sge.py

```diff
@@ -56,8 +58,7 @@ def __init__(self, queue=None, project=None, resource_spec=None, walltime=None,
 
         super(SGECluster, self).__init__(**kwargs)
 
-        header_lines = ['#!/bin/bash']
-
+        header_lines = ['#!/usr/bin/env bash']
```

Comment:
So you don't have a solution for propagating JOB_ID var in sge script?

Reply:
good catch. I thought I did but I'll have to sort it out.

Reply:
I take that back. SGE uses `JOB_ID` for its environment variable so its already present.